app/views/posts/index.html.erb

```diff
@@ -1 +1,51 @@
-<h1>this is the posts main page</h1>
\ No newline at end of file
+<h2>
+  <%= link_to "Go to all users", "/users", class: "button" %>
+</h2>
+<section class="user-profile">
+<div class="d-flex row justify-content-center mt-5">
```

Comment:
Please use a method from the post model in `app/views/posts/index.html.erb` to show the five most recent comments on a post.

Reply:
Added functionality to display five most recent comments for each post